src/graph.cc

```diff
@@ -614,6 +628,113 @@ bool ImplicitDepLoader::ProcessDepfileDeps(
   return true;
 }
 
+
+bool ImplicitDepLoader::LoadDynOutFile(State* state, DiskInterface* disk_interface, Edge* edge, const string& path,
+                                       vector<Node*>* nodes, int* outputs_count, string* err) {
```

Comment:
I have moved the parser code into `dynout_parser.cpp`, it did not make sense to have it in `graph.cc` but only used in `build.cc`. I am not using `re2c` because the parsing is for now very straightforward. It is just looking for new line.